### PR TITLE
OBS Studio add parameter for no desktop shortcut.

### PR DESCRIPTION
**Pull request by TheCakeIsNaOH**

### CODE REVIEW

**Comment by AeliusSaionji** [comment]
Have you tested this? It doesn't seem to be working.

**Comment by TheCakeIsNaOH** [comment]
Sorry about that, I must have reverted the change needed to the file location lines. Should be fixed.

File: obs-studio.install/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ﻿$ErrorActionPreference = 'Stop';
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
-$fileLocation   = Get-Item $toolsDir\*_x32.exe
-$file64Location = Get-Item $toolsDir\*_x64.exe
+$fileLocation   = Join-Path $toolsDir (Get-childitem -Path $toolsDir -Filter *x86.exe -Name)
+$file64Location = Join-Path $toolsDir (Get-childitem -Path $toolsDir -Filter *x64.exe -Name)
```
Comment (AeliusSaionji) [inline obs-studio.install/tools/chocolateyinstall.ps1]: Why is this changed? I'm not opposed to improvement but I don't know why you did this.

Comment (TheCakeIsNaOH) [reply obs-studio.install/tools/chocolateyinstall.ps1]: Unfortunately, `Get-Item` conflicts with the `Get-PackageParameters` that gets the parameter(s). `Get-childitem` works, but only gives the name of the file, so it needs to be joined with the rest of the path.

**Comment by TheCakeIsNaOH** [comment]
Hey, @AeliusSaionji are you planning on merging this?
